qutebrowser/misc/guiprocess.py

```diff
@@ -62,10 +64,11 @@ class GUIProcess(QObject):
     started = pyqtSignal()
 
     def __init__(self, what, *, verbose=False, additional_env=None,
-                 parent=None):
+                 parent=None, output_to_tab=False):
         super().__init__(parent)
         self._what = what
         self.verbose = verbose
+        self.output_to_tab = output_to_tab
```

Comment:
Since this isn't used outside of `GUIProcess`, it should be "private", i.e. prefixed with a `_`.